liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -1708,8 +1707,8 @@ protected void runLibertyMojoInstallFeature(Element features, String containerNa
      * @throws MojoExecutionException
      */
     @Override
-    protected void runLibertyMojoGenerateFeatures(Element classFiles) throws MojoExecutionException {
-        super.runLibertyMojoGenerateFeatures(classFiles);
+    protected void runLibertyMojoGenerateFeatures(Element classFiles, Boolean optimize) throws MojoExecutionException {
```

Comment:
Why does this need to be a `Boolean` rather than `boolean`?  In `StartDebugMojoSupport` you could just convert the `boolean` to a String without needing this intermediate object.

Reply:
That would be simpler. I will update that.